fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflow.java

```diff
@@ -44,6 +47,12 @@
 @Singleton
 public class DruidWorkflow implements RequestWorkflowProvider {
 
+    private static final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
+    private static final int DRUID_UNCOVERED_INTERVAL_LIMIT = SYSTEM_CONFIG.getIntProperty(
```

Comment:
If you make this non-static you can test behavior under different configurations, thus applying specific limits.